static/js/containers/CourseSearchPage.js

```diff
@@ -298,6 +305,10 @@ export class CourseSearchPage extends React.Component<Props, State> {
             <Cell width={4} key={i}>
               <SearchResult
                 result={result}
+                overrideObject={
```

Comment:
I think this approach is fine for now, but at some point in the future, it might be good to include the favorited status directly in the search index using the approach suggested by @rhysyngsun [here](https://github.com/mitodl/open-discussions/pull/2064#issuecomment-504550138) and [here](https://github.com/mitodl/open-discussions/pull/2064#issuecomment-504560461).   Mind if I open a tech-debt issue for this?

Reply:
yeah, my impression was that the general plan was to take this approach for now and refactor over to that more long-term solution later. so I think it's a good idea to open a tech debt issue. @rhysyngsun is that your impression too?

Reply:
I think a tech debt issue is fine, but I think we'd need a different approach than the one I previously suggested in what Matt linked since it significantly complicates reindexing. I can open an issue for it and I'll document previous discussion plus the new direction I have in mind.

Reply:
Issue: https://github.com/mitodl/open-discussions/issues/2142